src/oracledb_datapump/directives.py

```diff
@@ -306,18 +311,18 @@ def metadata(self) -> dict:
         return {"as_of": f"SCN({self.value})"}
 
 
-class FlashbackTime(ParameterTypeWithUserArgs[str], name="FLASHBACK_TIME"):
+class FlashbackTime(ParameterTypeWithUserArgs[datetime], name="FLASHBACK_TIME"):
     def __init__(self, value: str | datetime):
-        # arg: UTC datetime
         dt = try_parse_dt(value)
 
         if not isinstance(dt, datetime):
-            raise ValueError(f"Invalid UTC datetime: {value}")
-        self.value = "to_utc_timestamp_tz('{}')".format(dt.isoformat())
+            raise ValueError(f"Invalid datetime: {value}")
```

Comment:
Seems like it would be cleaner to wrap `try_parse_dt()` with a `try`/`except`. It's already throwing the exception so why not use it. Thoughts?

Reply:
Yeah `try_parse_dt` actually swallows the exception if a non-iso format string is passed to it. Its original use-case might have been a little different than its current usage, but I think I will revise to throw the exception within `try_parse_dt`.